src/test/java/com/svydovets/bibirnate/mapper/EntityMapperServiceTest.java

```diff
@@ -0,0 +1,159 @@
+package com.svydovets.bibirnate.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+
+import java.math.BigDecimal;
+import java.nio.charset.Charset;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mockito;
+
+import com.svydovets.bibirnate.entities.AbstractEntity;
+import com.svydovets.bibirnate.entities.AllTypesEntity;
+import com.svydovets.bibirnate.entities.ConstructorEntityWithException;
+import com.svydovets.bibirnate.entities.EntityNoDefaultConstructor;
+import com.svydovets.bibirnate.entities.EntityPrimitives;
+import com.svydovets.bibirnate.entities.EntityWrongType;
+import com.svydovets.bibirnate.exceptions.EntityMappingException;
+
+class EntityMapperServiceTest {
+
+    private ResultSet rs;
+    private EntityMapperService entityMapperService;
+
+    @BeforeEach
+    void beforeEach() {
+        rs = Mockito.mock(ResultSet.class);
+        entityMapperService = new EntityMapperService();
+    }
+
+    @Test
+    void mapToObjectSuccess() throws SQLException {
+        setupMocks(rs);
+        var result = entityMapperService.mapToObject(AllTypesEntity.class, rs);
+        Mockito.verify(rs, times(15)).getObject(anyString());
+        assertEquals(1234L, result.getId());
```

Comment:
u have a lot of repeated values that shuold be moved to constants. magic ints, strings etc. is not good practice